workspace/workspace_environment.go

```
@@ -47,17 +47,27 @@ func detectMcdrEnvironment(dir string, env *types.EnvironmentInfo) {
 		return
 	}
 
-	bytes, err := exec.Command("mcdreforged", "--version").Output()
+	for i, pluginDir := range config.PluginDirectories {
+		if !filepath.IsAbs(pluginDir) {
+			config.PluginDirectories[i] = filepath.Join(dir, pluginDir)
+		}
+	}
+
+	version := types.VersionUnknown
+	output, err := exec.Command("mcdreforged", "--version").Output()
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)